mappings/net/minecraft/entity/mob/MobEntity.mapping

```diff
@@ -27,7 +27,7 @@ CLASS ahf net/minecraft/entity/mob/MobEntity
 	METHOD A playAmbientSound ()V
 	METHOD D getAmbientSound ()Lxm;
 	METHOD F getLootTableId ()Lqc;
-	METHOD G ()Z
+	METHOD G cannotDespawn ()Z
```

Comment:
Does this get if it cannot despawn or set it so it cannot despawn?

Reply:
You definitely know this is a getter. A setter should be named `setNoDespawn` here instead.
`canXxx` `isXxx` are regular getter method name patterns, while `setXxx` is the setter.
`canXxx` is a verb phrase, so it is a good method name.